source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -205,11 +212,11 @@ The following code uses the ``InsertMany()`` method to insert four new
 ``Restaurant`` documents into a collection with
 ``BypassDocumentValidation`` set to ``true``:
 
-.. literalinclude:: /includes/fundamentals/code-examples/crud/insert.cs
+.. literalinclude:: /includes/code-examples/insert-many/InsertMany.cs
```

Comment:
It looks like this change removes the line that adds the `InsertManyOptions` which the example is meant to show. We should add that into the InsertMany file if we're going to use that instead of a separate insert code file

Reply:
Added the option to the InsertMany file.